lib/jekyll/scholar/version.rb

```diff
@@ -1,5 +1,9 @@
 module Jekyll
   class Scholar
+<<<<<<< HEAD
```

Comment:
Please remove the changes to the version file. Looks like you committed an unresolved conflict there.